src/System.IO.FileSystem/tests/File/Move.cs

```diff
@@ -144,7 +144,42 @@ public void FileNameWithSignificantWhitespace()
         }
 
         [Fact]
-        public void LongPath()
+        [PlatformSpecific(PlatformID.Windows)]
+        public void LongPath_Windows()
```

Comment:
This test seems to be testing a bunch of different things.  I think it'd be better to split it up.